monkey/infection_monkey/network/info.py

```diff
@@ -16,6 +20,10 @@
     long = int  # Python 3
 
 
+# Timeout for monkey connections
+TIMEOUT = 15
```

Comment:
I think we have different timeouts all over the place. I can think of ones where exploiting, where communicating with the Island. Do we need another one? Any way to consolidate?

Reply:
It's not another one, it's the one that was on control.py and was just moved here for dependency reasons.